CorpusSearch/Dockerfile

```diff
@@ -0,0 +1,33 @@
+﻿FROM mcr.microsoft.com/dotnet/aspnet:8.0 AS base
+RUN apt-get update && apt-get upgrade -y && \
+    apt-get install -y nodejs \
+    npm  
+USER $APP_UID
```

Comment:
Could you link to the docs? This is new from the .net8 images

Reply:
Here is a link: https://learn.microsoft.com/en-us/dotnet/core/compatibility/8.0
I only did this because Rider seemed to require it, wouldn't give me the Dockerfile without net8.0 and then when I tried to go back to net6.0, even in the terminal, it didn't quite work

Reply:
Or did you want the link somewhere in the code as well, as in the Dockerfile itself maybe

Reply:
In the code - this is for 'future David'